tests/bdd/test_csi_identity.py

```diff
@@ -8,22 +8,21 @@
 
 import pytest
 import docker
-import common
 import subprocess
 import csi_pb2 as pb
-import csi_pb2_grpc as rpc
-import grpc
 
-from common import CsiHandle
+from common.csi import CsiHandle
+from common.deployer import Deployer
+from common.apiclient import ApiClient
 
 
 @pytest.fixture(scope="module")
 def setup():
-    common.deployer_stop()
-    common.deployer_start(1)
+    Deployer.stop()
```

Comment:
Nit: Do we need this stop? Will the start just restart it? Plus, if all goes well, we call stop after the yield anyway.

Reply:
No, I don't think we don't, I'll remove it :+1: